src/main/java/gregtech/api/recipes/builders/FuelRecipeBuilder.java

```diff
@@ -0,0 +1,36 @@
+package gregtech.api.recipes.builders;
+
+import gregtech.api.recipes.Recipe;
+import gregtech.api.recipes.RecipeBuilder;
+import gregtech.api.recipes.RecipeMap;
+import gregtech.api.util.ValidationResult;
+
+public class FuelRecipeBuilder extends RecipeBuilder<FuelRecipeBuilder> {
+
+    public FuelRecipeBuilder() {
+
+    }
+
+    public FuelRecipeBuilder(Recipe recipe, RecipeMap<FuelRecipeBuilder> recipeMap) {
+        super(recipe, recipeMap);
+    }
+
+    public FuelRecipeBuilder(RecipeBuilder<FuelRecipeBuilder> recipeBuilder) {
+        super(recipeBuilder);
+    }
+
+    @Override
+    public FuelRecipeBuilder copy() {
+        return new FuelRecipeBuilder(this);
+    }
+
+    @Override
+    public FuelRecipeBuilder EUt(int EUt) {
+        return super.EUt(EUt * -1);
```

Comment:
Shouldn't this have a safety check for if `EUt` is negative? Something like `super.EUt(Math.abs(EUt) * -1)`